README.MD

```diff
@@ -91,11 +91,17 @@ go-hash
 To create a new database in a custom location, just provide the path to the new file for the database:
 
 ```
-go-hash /path/to/file
+go-hash -db /path/to/file
 ```
 
 If the file already exists, go-hash will try to load it as an existing database.
 
+`go-hash` will prompt for master password if idle for 60 seconds (default).  To change the timeout, use flag `-idle <number of seconds>`.  Use `0` for no timeout.
```

Comment:
Thanks for adding that, I think it's quite nice to let users decide where they want to stand on the security VS convenience trade-offs.